test/e2e/tests/test_root_ca.py

```diff
@@ -0,0 +1,167 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Integration tests for the ACMPCA Certificate resource.
+"""
+
+import time
+import logging
+import base64
+import pytest
+
+from acktest.resources import random_suffix_name
+from acktest.k8s import resource as k8s
+from acktest.k8s.resource import _get_k8s_api_client
+from kubernetes import client
+from e2e import service_marker, CRD_GROUP, CRD_VERSION, load_acmpca_resource
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.tests.helper import ACMPCAValidator
+from e2e.fixtures import k8s_secret
+
+RESOURCE_PLURAL = "certificates"
+
+CREATE_WAIT_AFTER_SECONDS = 10
+DELETE_WAIT_AFTER_SECONDS = 10
+
+@pytest.fixture(scope="module")
+def create_secret(k8s_secret):
+    secret = k8s_secret(
+        "default",
+        "certificate-secret",
+        "certificate",
+        "value"
+    )
+    yield secret
+
+@pytest.fixture(scope="module")
+def test_create_ca(acmpca_client):
+    ca_name = random_suffix_name("certificate-authority", 50)
+    replacements = {}
+    replacements["NAME"] = ca_name
+    replacements["COMMON_NAME"] = "www.example.com"
+    replacements["COUNTRY"] = "US"
+    replacements["LOCALITY"] = "Arlington"
+    replacements["ORG"] = "Example Organization"
+    replacements["STATE"] = "Virginia"
+
```

Comment:
oh looks like the idempotency token is not required.. we can only ignored then. no need to replace it with a random string

Reply:
Fixed this in commit "Remove IssueCertificateInput.IdempotencyToken"